.github/workflows/promote_to_staging.yml

```diff
@@ -1,18 +1,18 @@
-name: Deploy staging
+name: `Staging` environment deployment
 
 on:
   push:
     branches: [ staging ]
 
 env:
   environment: staging
-  credentials: ${{ secrets.AZURE_DIGITAL_TEST }}
+  credentials: ${{ secrets.AZURE_DIGITAL_DEV }}
 
   # From
-  registry_from: ${{ secrets.ACR_REGISTRY_TEST }}
-  registry_from_username: ${{ secrets.ACR_USERNAME_TEST }}
-  registry_from_password: ${{ secrets.ACR_PASSWORD_TEST }}
-  tag_from: test
+  registry_from: ${{ secrets.ACR_REGISTRY_DEV }}
+  registry_from_username: ${{ secrets.ACR_USERNAME_DEV }}
+  registry_from_password: ${{ secrets.ACR_PASSWORD_DEV }}
+  tag_from: dev
```

Comment:
should this be staging? or dev?

Reply:
Should be `Staging` have updated.

Reply:
Should be `dev` as that's where it will tag from. So `dev` => `staging` => `production`